src/Discord.Net.WebSocket/Entities/Messages/SocketUserMessage.cs

```diff
@@ -150,5 +150,13 @@ public Task ModifySuppressionAsync(bool suppressEmbeds, RequestOptions options =
         
         private string DebuggerDisplay => $"{Author}: {Content} ({Id}{(Attachments.Count > 0 ? $", {Attachments.Count} Attachments" : "")})";
         internal new SocketUserMessage Clone() => MemberwiseClone() as SocketUserMessage;
+
+        public async Task PublishAsync(RequestOptions options = null)
+        {
+            if (Channel.GetType() == typeof(SocketNewsChannel))
```

Comment:
Same as above for `RestUserMessage` (and I would probably reallocate this for the sake of aesthetics :p)

Reply:
Fixed and moved! :)